plan/physical_plans.go

```diff
@@ -185,17 +185,17 @@ type PhysicalTopN struct {
 
 // PhysicalApply represents apply plan, only used for subquery.
 type PhysicalApply struct {
-	basePhysicalPlan
+	physicalSchemaProducer
 
-	PhysicalJoin PhysicalPlan
+	PhysicalJoin *PhysicalHashJoin
```

Comment:
The `Apply` operator could only use HashJoin?

Reply:
yes